ElectronClient/gui/KeymapConfig/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import styles_ from './styles';
+import { ShortcutRecorder } from './ShortcutRecorder';
+const CommandService = require('lib/services/CommandService').default;
```

Comment:
Please always use `import` when importing a TypeScript file into another TypeScript file to benefit from type checking.